.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,67 +1,72 @@
-[Short description explaining the high-level reason for the pull request]
+_[Enter an explanation of what the pull request does and why.]_
 
-## Additions
 
--
-
-## Removals
-
--
+---
 
-## Changes
+_[Feel free to delete any checkboxes that are not applicable to this PR.]_
 
--
 
-## Testing
+## How to test this PR
 
 1.
 
+
 ## Screenshots
 
 
-## Notes
+## Notes and todos
 
 -
 
-## Todos
-
--
 
 ## Checklist
 
+_[Feel free to delete any checkboxes that are not applicable to this PR.]_
+
 - [ ] PR has an informative and human-readable title
+  - PR titles are used to generate the change log in [releases](../../releases),
+    so good ones make that easier to scan.
+  - Consider prefixing with the feature/section you're addressing,
+    e.g., "Mega Menu: fix layout bug" or "Paying for College: Update content on Repay tool"
 - [ ] Changes are limited to a single goal (no scope creep)
-- [ ] Code can be automatically merged (no conflicts)
 - [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
-- [ ] Passes all existing automated tests
-- [ ] Any _change_ in functionality is tested
-- [ ] New functions are documented (with a description, list of inputs, and expected output)
-- [ ] Placeholder code is flagged / future todos are captured in comments
-- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
+- [ ] Future todos are captured in comments and/or tickets
 - [ ] Project documentation has been updated
-- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:
+  - Potential places for documentation:
+    - [This repo’s docs](https://cfpb.github.io/cfgov-refresh/) (edit the files in the `/docs` folder) –
+      for basic, close-to-the-code documentation on working with this repo
+    - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
+    - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
```

Comment:
Here I added my own attempt to briefly describe what type of docks should go where. Open to discussion about whether or not what I've written seems accurate.

Reply:
September regroup: "One doc to rule them all!" 😐

Reply:
I don't know who you're quoting, but I've personally never believed that is a realistic goal :)

Reply:
It was one of the post-it note clusters at the wrap-up session.